src/app_service/service/message/service.nim

```diff
@@ -31,6 +31,7 @@ const SIGNAL_MESSAGES_MARKED_AS_READ* = "messagesMarkedAsRead"
 const SIGNAL_MESSAGE_REACTION_ADDED* = "messageReactionAdded"
 const SIGNAL_MESSAGE_REACTION_REMOVED* = "messageReactionRemoved"
 const SIGNAL_MESSAGE_DELETION* = "messageDeleted"
+const SIGNAL_MESSAGE_EDITED* = "new-messageEdited"
```

Comment:
@Khushboo-dev-cpp please remove "new-" prefix here, no need to have it any more since we completely moved to using new code, there is no mixture of "new" and "old" signals any more in the app.

Reply:
I missed that. removed now :)